stories/IsolatedBlockEditor.stories.tsx

```diff
@@ -12,6 +18,62 @@ export const Default = () => {
 	return <IsolatedBlockEditor settings={ {} } />;
 };
 
+export const Managed = ( { onInput, onChange, onUndo, onRedo } ) => {
+	const [ blocks, setBlocks ] = useState( [] );
+	const [ log, setLog ] = useState( [] );
+
+	useEffect( () => {
+		registerCoreBlocks();
```

Comment:
Just curious: I would assume that since we're rendering the `<IsolatedBlockEditor>` component, the [initialization code](https://github.com/Automattic/isolated-block-editor/blob/1b5c9a8c0f04aab2c2961a2b3f19e0619e365410/src/index.js#L208) including this core block registration should be handled internally. Is this not the case?

Reply:
by "handled internally" do you mean performing this `useEffect` inside the `initializeIsoEditor` function? If so, yes, I think it makes sense moving it over there.

Reply:
I meant that simply rendering `<IsolatedBlockEditor>` should already be triggering the `registerCoreBlocks()` function here:

https://github.com/Automattic/isolated-block-editor/blob/1b5c9a8c0f04aab2c2961a2b3f19e0619e365410/src/index.js#L127

So I was wondering if the call in the `useEffect` is redundant.